plugins/rikki/heroeslounge/plugin.yaml

```diff
@@ -25,6 +26,7 @@ navigation:
                 icon: icon-tag
                 permissions:
                     - rikki.heroeslounge.season
+                    - rikki.heroeslounge.individual_playoffs
```

Comment:
For my own understanding: Access is granted for users with any of these, right?

Reply:
Yes